core/lib/dal/src/models/storage_block.rs

```diff
@@ -625,3 +625,17 @@ impl ResolvedL1BatchForL2Block {
         self.block_l1_batch.unwrap_or(self.pending_l1_batch)
     }
 }
+
+pub struct StoragePubdataParams {
```

Comment:
Nit: Please restrict visibility to `pub(crate)` to make unused fields (should they occur in the future) more visible.